openwisp_network_topology/tests/test_admin.py

```diff
@@ -133,7 +133,7 @@ def test_custom_topology_receive_url(self):
         t.save()
         path = reverse('{0}_topology_change'.format(self.prefix), args=[t.pk])
         # No change in URL Test
-        receive_path = f'topology/{t.pk}/receive/'
+        receive_path = f'topology/topology/{t.pk}/receive/'
```

Comment:
Seems we'll have to flag this patch as a backward incompatible change, but it looks needed to achieve consistency.